### PR TITLE
Document labels

### LINKED ISSUE #539: Document which labels are attached to things and when and why
probably in concepts under "observability"?

### PR DESCRIPTION
**Pull request by fhennig**
closes #539 

preview link: https://deploy-preview-540--stackable-docs.netlify.app/home/nightly/concepts/labels

### CODE REVIEW

File: modules/concepts/pages/labels.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Labels are key/value pairs in the metadata of Kubernetes objects that add identifying information to the object.
+They do not have direct semantic implications but can be used to organize and manage resources.
+stackablectl, the cockpit and the Helm Charts add labels to the resources that are part of a Stacklet, and the operators add labels to the resources they create.
+
+== Resource labels added by the operators
+
+Every resource created by a Stackable operator is labelled with a common set of labels.
+Some of these labels are https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/[recommended] to use by the Kubernetes documentation.
+These are the labels that are added:
```
Comment (Techassi) [inline modules/concepts/pages/labels.adoc:11]: suggestion: Slight rewording.

```suggestion
The following labels are added to resources:
```

File: modules/concepts/pages/labels.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
+= Labels
+
+Labels are key/value pairs in the metadata of Kubernetes objects that add identifying information to the object.
+They do not have direct semantic implications but can be used to organize and manage resources.
+stackablectl, the cockpit and the Helm Charts add labels to the resources that are part of a Stacklet, and the operators add labels to the resources they create.
```
Comment (Techassi) [inline modules/concepts/pages/labels.adoc:5]: suggestion: Add inline code block.

```suggestion
The `stackablectl` tool, the cockpit and the Helm Charts add labels to the resources that are part of a Stacklet, and the operators add labels to the resources they create.
```

File: modules/concepts/pages/labels.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+== Labels added by tools
+
+The resources associated with an operator installation are also labeled (or can be).
+
+=== stackablectl
+
+==== Installing operators
+
+When installing operators, stackablectl will always label the Deployment, ConfigMap, ClusterRole etc. with
```
Comment (Techassi) [inline modules/concepts/pages/labels.adoc:33]: suggestion: Use inline code block.

```suggestion
When installing operators, `stackablectl` will always label the Deployment, ConfigMap, ClusterRole etc. with
```

File: modules/concepts/pages/labels.adoc [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+When installing a stack or a demo, `stackablectl` adds additional labels to identify the parts of the stack or demo:
+
+* `stackable-tech/stack` with the value being the Stack name.
+* `stackable-tech/demo` with the value being the Demo name.
+
+These labels are attached to all parts of the Stack or demo directly installed by Stackable.
+This includes the operators that are needed, additional depdencies installed via Helm and additional required Kubernetes objects like Secrets or ConfigMaps.
+
+NOTE: The Stacklets themselves are not yet labeled with the Stack or Demo that they belong to.
```
Comment (Techassi) [inline modules/concepts/pages/labels.adoc:48]: note: We should clarify that _external_ resources, like Grafana, MinIO, etc. are also not labelled during installation as part of a demo or stack.

File: modules/concepts/pages/labels.adoc [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+
+=== Helm
+
+The Helm Charts for the Stackable operators support specifying labels in the `values.yaml` file.
+All resources deployed by the Helm Chart (Deployment, ClusterRole, Configmap etc.) will have the labels attached when the operator is deployed.
```
Comment (Techassi) [inline modules/concepts/pages/labels.adoc:54]: typo: Add comma.

```suggestion
All resources deployed by the Helm Chart (Deployment, ClusterRole, Configmap, etc.) will have the labels attached when the operator is deployed.
```

**Review by Techassi — CHANGES_REQUESTED** [review]

File: modules/concepts/pages/labels.adoc (hunk 4 above)
Comment (fhennig) [reply modules/concepts/pages/labels.adoc]: How about this? https://github.com/stackabletech/documentation/pull/540/commits/fee4f250cd4a776aecec56ae2b9b907b0aa53f5e

Comment (Techassi) [reply modules/concepts/pages/labels.adoc]: Yes, that should clarify it!

**Review by Techassi — APPROVED** [review]

**Pull request merged**